pkg/util/os/exec.go

```diff
@@ -0,0 +1,84 @@
+package os
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"sync"
+)
+
+// ExecInSystemWithParams can exec a command with some params in system.
+// All logs produced by command would be print to stdout and write into logsBuffer if it is not nil
+func ExecInSystemWithParams(execPath string, params []string, logsBuffer *bytes.Buffer, print bool) error {
+	paramStr := strings.Join(params, " ")
+	return ExecInSystem(execPath, paramStr, logsBuffer, print)
+}
+
+// ExecInSystem can exec a command with a full command in system.
+// All logs produced by command would be print to stdout and write into logsBuffer if it is not nil
+func ExecInSystem(execPath string, fullCommand string, logsBuffer *bytes.Buffer, print bool) error {
```

Comment:
I don't see a problem with this code, but someone at #583 made some comments earlier and wasted a lot of time over it.

cc @IronCore864

Reply:
Here there isn't a problem.

> "Both Command and CommandContext have built-in protections that will not let command arguments cause trouble. But make sure that the command itself is not controlled by the user, also do not use sh, because internal protection will not work in this case."

See more details here: https://semgrep.dev/docs/cheat-sheets/go-command-injection/.